ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java

```diff
@@ -117,7 +118,7 @@ public String getStatement() {
       justification = "overwriteProperties is unmodifiableMap()"
   )
   public Map<String, Object> getOverwriteProperties() {
-    return overwriteProperties;
+    return PropertiesUtil.coerceTypes(overwriteProperties, true);
```

Comment:
Just to clarify this is the actual and only fix of this PR right? All others seems to be cleanups with no logical changes.

Reply:
Also why we would like to ignore PropertyNotFoundException in command, while not in the ksql request below?

Reply:
The statement should already have been validated at this point, and we've committed to executing it. We should just deserialize and pass it along to the engine.

Reply:
Ah you're right!